tornjak-frontend/src/components/cluster-create.tsx

```diff
@@ -260,19 +331,19 @@ class ClusterCreate extends Component {
                   onChange={this.onChangeClusterManagedBy}
                 />
               </div>
-              <div className="agents-multiselect">
-                <MultiSelect.Filterable
+              <div className="agents-multiselect" data-test="agents-multiselect">
+                <FilterableMultiSelect
                   titleText="Assign Agents To Cluster"
                   helperText="i.e. spiffe://example.org/agent/myagent1..."
                   placeholder={this.state.agentsListDisplay}
-                  ariaLabel="selectors-multiselect"
+                  //ariaLabel="selectors-multiselect"
```

Comment:
hmm trying to understand the commented code here, is there a particular reason?

Reply:
Removed!! "araiaLabel" was just a label for the filterable multiselect. Which is taken care of by id here. And in typescript "araiaLabel" doesn't exist in the type.